### PR TITLE
Convert computed metrics to new format

### PR DESCRIPTION
**Pull request by TheConnMan**

### CODE REVIEW

**Review by codytipton — APPROVED** [review]

**Review by yadima — APPROVED** [review]

**Review by rclagett — APPROVED** [review]

File: analyticConfigurations/computed_metric-aws.sqs.json [hunk 1]
```diff
@@ -4,31 +4,43 @@
       {
         "match": "netuitive.aws.sqs.arrivalrate",
         "properties": {
-          "expression": "data['aws.sqs.numberofmessagessent'].actual / 300",
+          "expressions": [
+            "${aws.sqs.numberofmessagessent}.actual / 300",
+            "0"
```
Comment (jemassey) [inline analyticConfigurations/computed_metric-aws.sqs.json]: This wasn't in the original expression, it is equivalent to enabling sparse mode on this metric... prob not a good idea unless this was intentional?

File: analyticConfigurations/computed_metric-aws.sqs.json [hunk 2]
```diff
@@ -4,31 +4,43 @@
...
           "name": "Arrival Rate"
         }
       },
       {
         "match": "netuitive.aws.sqs.completionrate",
         "properties": {
-          "expression": "data['aws.sqs.numberofmessagesdeleted'].actual / 300",
+          "expressions": [
+            "${aws.sqs.numberofmessagesdeleted}.actual / 300",
+            "0"
```
Comment (jemassey) [inline analyticConfigurations/computed_metric-aws.sqs.json]: again here

File: analyticConfigurations/computed_metric-aws.sqs.json [hunk 3]
```diff
@@ -4,31 +4,43 @@
...
           "name": "Empty Receive Percent"
         }
       },
       {
         "match": "netuitive.aws.sqs.totalmessagesinqueue",
         "properties": {
-          "expression": "data['aws.sqs.approximatenumberofmessagesvisible'].actual + data['aws.sqs.approximatenumberofmessagesnotvisible'].actual + data['aws.sqs.approximatenumberofmessagesdelayed'].actual",
+          "expressions": [
+            "${aws.sqs.approximatenumberofmessagesvisible}.actual + ${aws.sqs.approximatenumberofmessagesnotvisible}.actual + ${aws.sqs.approximatenumberofmessagesdelayed}.actual",
+            "0"
```
Comment (jemassey) [inline analyticConfigurations/computed_metric-aws.sqs.json]: here also

**Review by jemassey — APPROVED** [review]
lgtm

**Review by jemassey — APPROVED** [review]
lgtm

**Comment by TheConnMan** [comment]
I have tested these new expressions.

**Pull request merged**